src/prefix.js

```diff
@@ -39,7 +39,11 @@ if (isInBrowser) {
   }
 
   // Correctly detect the Safari browser.
-  if (js === 'Webkit' && '-apple-trailing-word' in style) {
+  if (
+    js === 'Webkit' &&
+    ('-apple-trailing-word' in style ||
+      (typeof CSS !== 'undefined' && CSS.supports('-webkit-backdrop-filter', 'blur(1px)')))
```

Comment:
Why is this specific check a good technique to identify safari? Is it possible it will break in the future?

Reply:
@kof we didn't find any other way to detect safari correctly. Yes, there's the possibility to be on the same situation as now, because of CSS properties are dynamic thing. e.g. `-apple-trailing-word` will be removed soon - https://github.com/cssinjs/css-vendor/issues/83.

Reply:
maybe we shouldn't detect vendor at all in a generic way, but instead, look into those specific cases where it is used and detect support of those specific features!

Reply:
it's used in two cases:

1. https://github.com/cssinjs/css-vendor/blob/33bc7c729ce2c49524b80f7e7d3b2070cfe7acb0/src/plugins/text-orientation.js#L9
2. https://github.com/cssinjs/css-vendor/blob/8a86b6195b4e14a844d98aaf5a1c70f672e347d3/src/plugins/user-select.js#L9

In the first case, `-webkit-` should be applied to Safari (both desktop and iOS). According to caniuse - https://caniuse.com/#search=user-select.
With seconds case it's pretty much same - https://caniuse.com/#search=text-orientation.

The overall idea is good, but honestly, I don't know how to implement it 🤷‍♂️